src/app/hooks/addressLookUp.js

```diff
@@ -0,0 +1,27 @@
+const mbxClient = require("@mapbox/mapbox-sdk");
```

Comment:
Lets make a `util` folder for this to go in instead as it's not really a hook and could be confusing